### PR TITLE
Splitting Vector Databases into individual cookbooks

### PR DESCRIPTION
**Pull request by colin-openai**
This is the first stab at splitting these out individually - there is a common description for all, so it may be worth taking this out and putting it in the README.md instead.

Two still to be finalised:
- Milvus
- Zilliz

Four that do not have a Getting Started notebook but do have a folder:
- AnalyticDB
- Hologres
- Kusto
- SingleStoreDB

### CODE REVIEW

**Comment by ted-at-openai** [comment]
Awesome; thank you. I think this makes it way more readable.

Suggestion: do you think it makes sense to stick all of the `Using_X_for_embeddings_search.ipynb` into the folders for each vector DB provider?

File: examples/vector_databases/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Vector Databases
+
+This section of the cookbook showcases many of the vector databases available to support your semantic search use cases.
```
Comment (ted-at-openai) [inline examples/vector_databases/README.md:3]: suggestion: 'the cookbook' -> 'the OpenAI Cookbook'

File: examples/vector_databases/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Vector Databases
+
+This section of the cookbook showcases many of the vector databases available to support your semantic search use cases. 
+
+Vector databases can be a great accompaniment for knowledge retrieval applications, which reduce hallucinations by providing the LLM with the relevant context to answer their questions.
```
Comment (ted-at-openai) [inline examples/vector_databases/README.md:5]: suggestion: 'their' -> ''

**Comment by ted-at-openai** [comment]
^is the idea that these are all OpenAI-written whereas the ones in the folder are third-party-written?

(my instinct from a user POV is to put them all together in single folders to make it easier for people to find everything)

**Comment by colin-openai** [comment]
@ted-at-openai got it, moved them all into the named provider folders and updated the wording in the Readme. Can add more detail to the Readme but otherwise I think we're good to go.

**Comment by ted-at-openai** [comment]
Whoops, I merged an update by Zilliz which is blocking merging. Mind resolving? Otherwise if you think it's good to go, feel free to merge whenever you like.

**Comment by colin-openai** [comment]
@ted-at-openai solved merge issue, merging results

**Pull request merged**